tasks.py

```diff
@@ -187,40 +188,53 @@ def dev_env(ctx, architecture="amd64", name="kind", cni=None):
             tmp.flush()
             run("kind create cluster --name={} --config={}".format(name, tmp.name), pty=True, echo=True)
 
-    config = run("kind get kubeconfig-path --name={}".format(name), hide=True).stdout.strip()
-    env = {"KUBECONFIG": config}
     if mk_cluster and cni:
-        run("kubectl apply -f e2etest/manifests/{}.yaml".format(cni), echo=True, env=env)
+        run("kubectl apply -f e2etest/manifests/{}.yaml".format(cni), echo=True)
 
     build(ctx, binaries=["controller", "speaker", "mirror-server"], architectures=[architecture])
     run("kind load docker-image --name={} metallb/controller:dev-{}".format(name, architecture), echo=True)
     run("kind load docker-image --name={} metallb/speaker:dev-{}".format(name, architecture), echo=True)
     run("kind load docker-image --name={} metallb/mirror-server:dev-{}".format(name, architecture), echo=True)
 
     run("kubectl delete po -nmetallb-system --all", echo=True)
-    with open("manifests/metallb.yaml") as f:
-        manifest = f.read()
-    manifest = manifest.replace(":main", ":dev-{}".format(architecture))
-    manifest = manifest.replace("imagePullPolicy: Always", "imagePullPolicy: Never")
-    with tempfile.NamedTemporaryFile() as tmp:
-        tmp.write(manifest.encode("utf-8"))
-        tmp.flush()
-        run("kubectl apply -f {}".format(tmp.name), echo=True, env=env)
+
+    manifests_dir = os.getcwd() + "/manifests"
+    with tempfile.TemporaryDirectory() as tmpdir:
+        # Copy manifests dir to temp dir.
+        shutil.copytree(manifests_dir, tmpdir, dirs_exist_ok=True)
```

Comment:
`dirs_exist_ok` was introduced in python 3.8 (for example, python 3.8 was uploaded to debain unstable at the end of Nov 2019). The reference [here](https://docs.python.org/3/library/shutil.html) says it was introduced in 3.8

My local env fails because I'm using python 3.7, for example. I'd prefer to not force such a new python version for this to work. Also, taking into account commit 19cb437d12399e67d9b870b0675fd640f3ed0975 that added a workaround for python2/python3, it seems that this is an effort that we are already doing.

Can you please remove that argument and find a workaround that works with older python versions? :-) Not sure how old, I trust your judgement, but not "the latests python release only" would be great :)

Reply:
Thanks. Indeed I'm using Python 3.8 and wasn't aware this is a new flag. I'll fix this.

Reply:
Eventually I've opted for applying plain manifests (namespace, secret and MetalLB) rather than an entire directory. The reason is - this seems to be the main way for deploying MetalLB according to the [docs](https://metallb.universe.tf/installation/).

Reply:
I've verified the current code works with the following Python versions:

- `3.8.2`
- `3.7.4`
- `3.5.6`

Python 2 is EOL so I don't think we should worry about it.